src/main.js

```diff
@@ -33,9 +25,18 @@ let load = base.loader({
     }
   },
 
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
+      project: cfg.monitor.component,
```

Comment:
Yea, that's what's currently being used (separate project for staging and production) and I expect to give credentials to both that are not shared.